src/main/java/bit/javaoop/AbstractManager.java

```diff
@@ -0,0 +1,34 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+
+public abstract class AbstractManager extends AbstractEmployee{
+
+    ArrayList<AbstractEmployee> employees_list;
+
+    AbstractManager(String name, double salary, ArrayList<AbstractEmployee> employees_list) {
+        super(name, salary);
+        this.employees_list = new ArrayList<>();
+    }
+
+    ArrayList<AbstractEmployee> getEmployees_list() {
+        return employees_list;
+    }
+
+    public void setEmployees_list(ArrayList<AbstractEmployee> employees_list) {
+        this.employees_list = employees_list;
+    }
+
+
+    public String toStr(int line) {
+        String result = new String();
+        for(int i=0;i<line;i+=1)
+            result+='\t';
```

Comment:
If you use IntelliJ it will tell you that:

<img width="465" alt="screen shot 2018-03-26 at 21 14 35" src="https://user-images.githubusercontent.com/17063095/37927548-bc3370fe-313a-11e8-8655-d1c874ee4de7.png">

It means that you should use a `StringBuilder` instead. 
Strings in Java are implemented using a Flyweight design pattern (https://en.wikipedia.org/wiki/Flyweight_pattern) which means that every time you append something to a String it creates a new objected which makes it very inefficient in terms of memory.